src/main/java/com/minecolonies/coremod/util/TeleportToColony.java

```diff
@@ -0,0 +1,92 @@
+package com.minecolonies.coremod.util;
+
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.IColony;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Helper class for teleporting players to home or a friendly colony.
+ */
+public final class TeleportToColony
+{
+    private static final String CANT_FIND_COLONY = "No Colony found for teleport, please define one.";
+    private static final String CANT_FIND_PLAYER = "No player found for teleport, please define one.";
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private TeleportToColony(@NotNull final String... parents)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "TeleportToColony" constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)